src/test/integration/IntegrationBase.t.sol

```diff
@@ -1193,7 +1193,7 @@ abstract contract IntegrationBase is IntegrationDeployer, TypeImporter {
 
         // For each strategy, check (prev - removed == cur)
         for (uint i = 0; i < strategies.length; i++) {
-            assertEq(prevShares[i] - removedShares[i], curShares[i], err);
+            assertEq(prevShares[i], curShares[i] + removedShares[i], err);
```

Comment:
oof, i know exactly how annoying these can be to debug.

it feels very counterintuitive to write the assertions like this, though :sweat: